models/rating.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function(sequelize, DataTypes) {
+    var rating = sequelize.define("rating", {
+      beer_id: {
+        type: DataTypes.INTEGER,
+        allowNull: false
+      },
+      user_id: {
+        type: DataTypes.INTEGER,
+        allowNull: false
+      },
+      rating: {
+          type: DataTypes.FLOAT(2,1),
+          allowNull: false
+      }
+    });
+    return rating;
+  };
```

Comment:
Most likely not a big deal, but the GitHub review Ui is showing 'No newline at end of file'

```suggestion
  };
  
```
 I don't know what issues, if any this would cause, but might as well fix it

 Same error also at end of schema.sql and user.js files